UefiTestingPkg/AuditTests/PagingAudit/UEFI/PagingAuditCommon.c

```diff
@@ -342,8 +339,8 @@ MemoryAttributesTableDump (
   FormattedStringSize = AsciiSPrint (TempString, MAX_STRING_SIZE, MatFormatString, 0, 0, 0, 0, 0, NONE_GCD_MEMORY_TYPE);
   // Make sure to add space for the NULL terminator at the end.
   BufferSize = (EntryCount * FormattedStringSize) + sizeof (CHAR8);
-  Buffer     = AllocatePool (BufferSize);
-  if (!Buffer) {
+  Status     = gBS->AllocatePool (EfiBootServicesData, BufferSize, (VOID **)&Buffer);

```

Comment:
Is this change going to break the SMM driver?

Reply:
The SMM driver does not use PagingAuditComon.c. The SMM test app does, but it's a UEFI_APPLICATION which calls into the driver to collect SMM paging data.